R-notebooks/make.R

```diff
@@ -18,7 +18,7 @@ for (file_name in file_names) {
     )),
     error = function(e) {
       cat(e$message)
-      has_failures <<- TRUE
+      has_failures = TRUE
```

Comment:
This change is incorrect; `<<-` is necessary for errors to be correctly propagated, since with `=`, `has_failures` will be local to this anonymous function. For example:

```
> has_failures = FALSE
> tryCatch(stop("ducks!"), error = function(e) { has_failures = TRUE })
> has_failures
[1] FALSE
> tryCatch(stop("ducks!"), error = function(e) { has_failures <<- TRUE })
> has_failures
[1] TRUE
```

Suggested fix is just
```suggestion
      has_failures <<- TRUE
```